app/src/main/java/edu/drexel/lapcounter/lapcounter/frontend/DeviceSelectActivity.java

```diff
@@ -145,8 +155,8 @@ public void viewDevice(View view) {
         Intent intent = new Intent(this, DeviceInfoActivity.class);
         // TODO: use intent.putExtra() to pass information to the next activity
         intent.putExtra("DEVICE_NAME",mDevice.getName());
-        intent.putExtra("DEVICE_MAC",mDevice.getMac());
-        intent.putExtra("DEVICE_RSSI",mDevice.getRssi());
+        intent.putExtra("DEVICE_MAC",mDevice.getMacAddress());
+        intent.putExtra("DEVICE_RSSI",mDevice.getThreshold());
```

Comment:
Note for the future: these strings should be constants. like `EXTRAS_DEVICE_NAME` so the receiver can access the same string. 

Also, perhaps use `DEVICE_THRESHOLD` instead of `DEVICE_RSSI`